lib/mbta_v3_api/facilities/repo.ex

```diff
@@ -5,6 +5,18 @@ defmodule MBTAV3API.Facilities.Repo do
   use RepoCache, ttl: :timer.hours(1)
 
   alias MBTAV3API.Facilities
+  alias MBTAV3API.Facilities.Facility
+
+  def get(id, opts \\ []) when is_binary(id) do
+    case cache({id, opts}, fn {id, opts} ->
+           with %{data: [facility]} <- Facilities.get(id, opts) do
+             {:ok, facility}
+           end
+         end) do
+      {:ok, facility} -> Facility.parse(facility)
+      {:error, _} -> nil
```

Comment:
Is swallowing this error OK?

Reply:
It's what we're doing with the other exposed functions of this type, I didn't want this one to be different. We don't currently have a reason to care about the type of error when we consume it.